my-demo-project/model/src/main/java/Student.java

```diff
@@ -19,6 +17,9 @@ public Student(String name, int mathGrade, int economicGrade, int englishGrade,
         this.computerScienceGrade = computerScienceGrade;
     }
 
+    public Student() {
+    }
```

Comment:
Для SonarQube. Потом прикрутим lombok и упростим себе жизнь, но пока что пустые методы и конструкторы должны резать тебе глаз.
Кстати, не забывай в классы, которые используются в коллекциях, вкручивать equals и hashCode

```suggestion
    public Student() {
        //default constructor
    }
```